src/components/calculator/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './calculator.css';
+
+const Calculator = () => (
+  <div className="wrapper">
+    <div className="output">
+      0
+    </div>
+    <div className="keys">
+      <div className="ac">AC</div>
+      <div className="plus-minus">+/-</div>
+      <div className="percent">%</div>
+      <div className="division">/</div>
+      <div className="seven">7</div>
+      <div className="eight">8</div>
+      <div className="nine">9</div>
+      <div className="multiply">*</div>
+      <div className="four">4</div>
+      <div className="five">5</div>
+      <div className="six">6</div>
+      <div className="minus">-</div>
+      <div className="one">1</div>
+      <div className="two">2</div>
+      <div className="three">3</div>
+      <div className="plus">+</div>
+      <div className="zero">0</div>
+      <div className="dot">.</div>
+      <div className="equal">=</div>
+    </div>
+  </div>
```

Comment:
I really love the way you styled your app, it's cool and very pleasing, but; 
- [ ] According to the design guidelines, you're required to follow the design given and to match your design with that. Please modify your styles to match the original design. The goal is to train you to work on design guidelines given for a project as that's what awaits you in your future job. 👍🏽
 
<img width="582" alt="Screenshot 2023-02-21 at 8 53 18 PM" src="https://user-images.githubusercontent.com/50941074/220457283-4679b103-ad7c-40d5-bc0a-b0916671be9d.png">